### PR TITLE
feat: improve the Infobox style (manifest)

### PR DESCRIPTION
**Pull request by mimoham24**
# Overview

### CODE REVIEW

**Review by HideBa — CHANGES_REQUESTED** [review]
@mimoham24 Can you change the manifest_ja file as well?
I or @rot1024 will add the suggested change, but at least plz get the other part ready. 🙏

File: pkg/builtin/manifest.yml [hunk 1]
```diff
@@ -298,6 +298,50 @@ extensions:
...
+              choices:
+                - key: auto
+                  label: Auto
+                - key: manual
+                  label: Manual
+            - id: height
+              type: number
+              title: Height
+              #              min: 0
+              #              max: 70
```
Comment (HideBa) [inline pkg/builtin/manifest.yml:339]: Is there any reason why you left the comment here?

File: pkg/builtin/manifest_ja.yml [hunk 2]
```diff
@@ -152,29 +152,51 @@ extensions:
...
-          title: 余白右
-          description: "右端の余白を設定します。最小0px・最大40ox"
+        title: ディフォルト
         fields:
           title:
             title: タイトル
+          showTitle:
+            title: タイトル表示
+          position:
+            title: ポジション
```
Comment (HideBa) [inline pkg/builtin/manifest_ja.yml:162]:
```suggestion
            title: 位置
```

File: pkg/builtin/manifest_ja.yml [hunk 3]
```diff
@@ -152,29 +152,51 @@ extensions:
...
-        paddingBottom:
-          title: 余白下
-          description: "下端の余白を設定します。最小0px・最大40ox"
-        paddingLeft:
-          title: 余白左
-          description: "左端の余白を設定します。最小0px・最大40ox"
-        paddingRight:
-          title: 余白右
-          description: "右端の余白を設定します。最小0px・最大40ox"
+        title: ディフォルト
```
Comment (HideBa) [inline pkg/builtin/manifest_ja.yml:155]:
```suggestion
        title: デフォルト
```

File: pkg/builtin/manifest_ja.yml [hunk 4]
```diff
@@ -152,29 +152,51 @@ extensions:
...
+              middle: 中央
+              left: 左
           size:
             title: サイズ
             choices:
               small: 小
+              medium: 中
               large: 大
+          heightType:
+            title: 高さ種類
```
Comment (HideBa) [inline pkg/builtin/manifest_ja.yml:174]:
```suggestion
            title: 高さ設定
```

File: pkg/builtin/manifest_ja.yml [hunk 5]
```diff
@@ -152,29 +152,51 @@ extensions:
...
+            description: "上端の余白を設定します。最小0px・最大40ox"
+          infoboxPaddingBottom:
+            title: 余白下
+            description: "下端の余白を設定します。最小0px・最大40ox"
+          infoboxPaddingLeft:
+            title: 余白左
+            description: "左端の余白を設定します。最小0px・最大40ox"
+          infoboxPaddingRight:
+            title: 余白右
+            description: "右端の余白を設定します。最小0px・最大40ox"
```
Comment (HideBa) [inline pkg/builtin/manifest_ja.yml:191]:
```suggestion
          infoboxPaddingTop:
            title: 余白上
            description: "上端の余白を設定します。最小0px・最大40px"
          infoboxPaddingBottom:
            title: 余白下
            description: "下端の余白を設定します。最小0px・最大40px"
          infoboxPaddingLeft:
            title: 余白左
            description: "左端の余白を設定します。最小0px・最大40px"
          infoboxPaddingRight:
            title: 余白右
            description: "右端の余白を設定します。最小px・最大40px"
```

File: pkg/builtin/manifest_ja.yml [hunk 6]
```diff
@@ -152,29 +152,51 @@ extensions:
...
           bgcolor:
             title: 背景色
+          outlineWidth:
+            title: アウトライン幅
+          outlineColor:
+            title: アウトラインカラー
+          useMask:
+            title: マスクを使用する
```
Comment (HideBa) [inline pkg/builtin/manifest_ja.yml:199]:
```suggestion
          outlineWidth:
            title: 線幅
          outlineColor:
            title: 線色
          useMask:
            title: 背景オーバーレイ
```

**Review by HideBa — APPROVED** [review]
Added Japanese text

File: pkg/builtin/manifest.yml (hunk 1 above)
Comment (mimoham24) [reply pkg/builtin/manifest.yml:339]: @HideBa red said that he will provide me the min/max later ... that's why I guess
let me double-check with him and decide what to do with those two lines

**Review by HideBa — APPROVED** [review]
LGTM! 👍

**Pull request merged**